src/mono/sample/wasm/browser-bench/frame-main.js

```diff
@@ -3,7 +3,7 @@
 
 "use strict";
 
-import { dotnet, exit } from './_framework/dotnet.js'
+import { dotnet, exit } from './dotnet.js'
```

Comment:
I guess this will work because of the `WasmRuntimeAssetsLocation` change, right?

Reply:
Exactly